modules/widgets/twitter-timeline.php

```diff
@@ -290,6 +291,7 @@ public function form( $instance ) {
 			'theme'        => 'light',
 			'chrome'       => array(),
 			'tweet-limit'  => null,
+			'dnt'          => true,
```

Comment:
Could you add a filter here, so folks can choose to change that value to false later on?